logwatch/logwatch_user_loki_test.go

```diff
@@ -0,0 +1,55 @@
+package logwatch_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/logwatch"
+)
+
+/* These tests are for testing Loki format, they rarely change so you can run them manually */
+
+func TestExampleLokiStreaming(t *testing.T) {
+	t.Skip("uncomment and run manually")
+	t.Run("stream all container logs to Loki, subtest 1", func(t *testing.T) {
+		testData := testData{repeat: 1, perSecond: 0.01, streams: []string{"A\nB\nC\nD", "E\nF\nG\nH"}}
+		d, err := NewDeployment(testData)
+		// nolint
+		defer d.Shutdown()
```

Comment:
You're risking nil pointer errors here. Check error before calling shutdown.

Reply:
fixed